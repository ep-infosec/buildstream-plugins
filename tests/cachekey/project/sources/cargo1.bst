kind: import
sources:
- kind: local
  path: files/local
- kind: cargo
  url: upstream:crates
  ref:
  - name: foo
    version: 1.0
    sha: thisisthecratesha
