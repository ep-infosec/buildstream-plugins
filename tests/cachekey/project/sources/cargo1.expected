c58a12e23b6d497196fc724102d06f62c64a73cedc6e12b2b93c6bd1e70dc17f